.gitignore

```diff
@@ -3,3 +3,5 @@ terraform.tfstate*
 config-drive.iso
 venv
 *.pyc
+.terraform.lock.hcl
```

Comment:
```suggestion
```
You are supposed to commit lock files since they contain the exact version of any dependencies (modules, providers) that you used.